src/nvim/generators/gen_api_dispatch.lua

```diff
@@ -301,19 +305,19 @@ end
 output:write('\n}\n\n')
 output:close()
 
-mpack_output = io.open(mpack_outputf, 'wb')
+local mpack_output = io.open(mpack_outputf, 'wb')
 mpack_output:write(mpack.pack(functions))
 mpack_output:close()
 
-local function include_headers(output, headers)
-  for i = 1, #headers do
-    if headers[i]:sub(-12) ~= '.generated.h' then
-      output:write('\n#include "nvim/'..headers[i]..'"')
+local function include_headers(arg_output, arg_headers)
```

Comment:
I renamed it to "output_handle" is that fine? I couldn't think of a more appropriate name for the global output so didnt change it. Also named the arg_headers to "headers_to_include"